testing-modules/junit-5/src/test/java/com/baeldung/junit5/bean/test/NumbersBeanTest.java

```diff
@@ -0,0 +1,43 @@
+package com.baeldung.junit5.bean.test;
+
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.Test;
+
+import com.baeldung.junit5.bean.NumbersBean;
+
+/**
+ * Test class for {@link NumbersBean}.
+ * 
+ * @author Donato Rimenti
+ *
+ */
+public class NumbersBeanTest {
+
+    /**
+     * The bean to test.
+     */
+    private NumbersBean bean = new NumbersBean();
+
+    /**
+     * Tests that when an even number is passed to
+     * {@link NumbersBean#isNumberEven(int)}, true is returned.
+     */
+    @Test
```

Comment:
Better test names, using the givenX_whenY_thenZ formula, would be:
givenEvenNumber_whenCheckingIsNumberEven_thenTrue
givenOddNumber_whenCheckingIsNumberEven_thenFalse